integration/spark2/src/main/scala/org/apache/spark/sql/execution/strategy/CarbonLateDecodeStrategy.scala

```diff
@@ -388,9 +388,15 @@ private[sql] class CarbonLateDecodeStrategy extends SparkStrategy {
             }
           case other => other
       }
+      val updatedColumns = getRequestedColumns(relation,
```

Comment:
updatedProjects are not used in getRequestedColumn, why use it as a parameter and return value